frontend/src/service/Url.service.ts

```diff
@@ -179,9 +185,9 @@ export class UrlService {
   private gqlCreateURLVariable(
     captchaResponse: string,
     link: Url,
-    alias: string | null,
     isPublic: boolean = false
   ) {
+    let alias = link.alias === '' ? null : link.alias!;
```

Comment:
Please add a TODO to mention the backend change we have to make.

Reply:
Just saw the issue being opened. This one is resolved.

Reply:
Done